src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  vertical-align: baseline;
+  font-weight: inherit;
+  font-family: inherit;
+  font-style: inherit;
+  font-size: 100%;
+  border: 0 none;
+  outline: 0;
+  padding: 0;
+  margin: 0;
```

Comment:
morgin and paddings you should use directly to that element what you need.
for this selector (*) you should set only box-sizing